README.md

```diff
@@ -66,10 +66,10 @@ nvm is a version manager for [node.js](https://nodejs.org/en/), designed to be i
 
 To **install** or **update** nvm, you should run the [install script][2]. To do that, you may either download and run the script manually, or use the following cURL or Wget command:
 ```sh
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+bash -c "$(curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh)"
 ```
 ```sh
-wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+bash -c "$(wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh)"
```

Comment:
(we probably discussed this on the other PR, but) why is this needed? there's nothing wrong with these commands as-is.

Reply:
I changed them to unify with Ansible.

Reply:
Actually, there is a risk that if the `curl`/`wget` fail, the whole pipe will not. The option is to do `(set -o pipefail && curl -o- ... | bash)`. Which could also be done in Ansible as
```yaml
shell: |
  set -o pipefail
  curl -o- ... | bash
```

Then ansible-lint will be happy as well.

Reply:
The pipe will fail in that case because an invalid shell script will be piped to bash, so there's really not a risk there.

I'd rather use ignore comments, than change to something unnecessary merely to satisfy a linter. Does ansible-lint have that feature?